docs/NativeModulesAndroid.md

```diff
@@ -89,21 +89,40 @@ ReadableArray -> Array
 ### Register the Module
 
 The last step within Java is to register the Module; this happens in the `createNativeModules` of your apps package. If a module is not registered it will not be available from JavaScript.
-
+In the package class it is important you override three methods: `createNativeModules`, `createJSModules` and `createViewManagers`.
 ```java
-class AnExampleReactPackage implements ReactPackage {
+//AnExampleReactPackage.java
+package com.example.modules.toast;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import com.facebook.react.ReactPackage;
+import com.facebook.react.bridge.JavaScriptModule;
+import com.facebook.react.bridge.NativeModule;
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.uimanager.ViewManager;
 
-  ...
+public class AnExampleReactPackage implements ReactPackage {
 
   @Override
   public List<NativeModule> createNativeModules(
                               ReactApplicationContext reactContext) {
-    List<NativeModule> modules = new ArrayList<>();
+    //Registering the module.
+    return Arrays.asList(new ToastModule(reactContext));
```

Comment:
This doesn't work for me, compilation fails with 
```
error: incompatible types: List<FlurryModule> cannot be converted to List<NativeModule>
        return Arrays.asList(new FlurryModule(reactContext));
```

I can compile with 
```java
        List<NativeModule> modules = new ArrayList<>();
        modules.add(new FlurryModule(reactContext));
        return modules;
```
Might be related to this: http://stackoverflow.com/a/14091803/1418983 ?
Although my module is not registered even when compilation works so I'm not sure.